### PR TITLE
Introduce lazily loaded feature modules

### PR DESCRIPTION
**Pull request by oliverguenther**
This PR refactors the routing modules currently contained all under work packages into a root routing module and several child modules which can depend on each other.

This allows us to lazily load the calendar module only when the route is accessed.

### CODE REVIEW

File: frontend/src/app/init-vendors.js [hunk 1]
```diff
@@ -82,5 +85,3 @@ require('expose-loader?URI!URIjs');
 require('URIjs/src/URITemplate');
 
 require("expose-loader?I18n!../vendor/i18n");
-
-require("fullcalendar/dist/locale-all.js");
```
Comment (ulferts) [inline frontend/src/app/init-vendors.js:86]: Is the goal then to split up the init-vendors into module specific inits in the long run?

File: frontend/src/app/modules/attachments/openproject-attachments.module.ts [hunk 2]
```diff
@@ -26,24 +26,18 @@
...
-import {AttachmentsUploadComponent} from 'core-components/attachments/attachments-upload/attachments-upload.component';
 import {NgModule} from '@angular/core';
-import {FormsModule} from '@angular/forms';
-import {BrowserModule} from '@angular/platform-browser';
+
 import {OpenprojectCommonModule} from 'core-app/modules/common/openproject-common.module';
+import {AttachmentsComponent} from "core-app/modules/attachments/attachments.component";
+import {AttachmentListComponent} from "core-app/modules/attachments/attachment-list/attachment-list.component";
+import {AttachmentListItemComponent} from "core-app/modules/attachments/attachment-list/attachment-list-item.component";
+import {AttachmentsUploadComponent} from "core-app/modules/attachments/attachments-upload/attachments-upload.component";
```
Comment (ulferts) [inline frontend/src/app/modules/attachments/openproject-attachments.module.ts:35]: :+1:

**Comment by ulferts** [comment]
I like that change and I also like that the overall size seems to have decreased (at least there are considerably less packages now).

File: frontend/src/app/init-vendors.js (hunk 1 above)
Comment (oliverguenther) [reply frontend/src/app/init-vendors.js:86]: Yes, but I didn't want to do too much while you're still on the grid. You can probably move the grid into its own module and include the calendar module there to ensure the embedded calendar component is loaded. Let's revisit this in a separate PR.

**Pull request merged**